### DISCUSSION TITLE
i need help😅

### DISCUSSION THREAD

**Discussion by randomguy213123** [DaBluLite/DiscordColorways#1] [General]
so, when i download the plugin, discord cant load it in its plugin folders and crashes my discord over and over again,help please😭

**Comment by randomguy213123** [comment]
oh btw im using virtual boy

**Reply by DaBluLite** [reply]
are you using betterdiscord?

**Comment by DaBluLite** [comment]
Forgot to close this, the BD plugin should work fine, for the most part
